DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: austraits
 Title: Helpful functions to access, summarise and wrangle austraits data
-Version: 1.0.0
+Version: 1.0.0.9000
```

Comment:
If you're merging into master, you first need to remove the .9000 and increment.  This would her version 1.1.0

Reply:
Ah yes, gets me everytime! Will change now! Woohoo! This is our first minor release! Woohoo!